package.json

```diff
@@ -20,6 +20,7 @@
     "test:unit": "mocha 'test/unit-tests/**/*.test.js' --timeout 6000 --trace-warnings",
     "test": "npm run test:api && npm run test:unit:silent",
     "test-functional": "cd test/functional-tests && npm install && npm test",
+    "test-docs": "cd test/docs-tests && node generateWebsite.js && cd ./gauge && npm install && npm test",
```

Comment:
Before merging I wanted to check where the test is run. I think we should run it after a check-in via GitHub actions. However I do not see a workflow entry for running it.

Reply:
Yes.
Currently you can run it manually like `npm run test-docs`, with no automation.

If I understand correctly, you are suggesting to insert in `.github/workflows/taiko` an additional job that launches this test (which I have never done, but I can try). Is my understanding correct?

Should we instead or in addition add an `husky` hook before committing or pushing?
Like 
 ```json
//package.json
...
 "husky": {
    "hooks": {
      "pre-commit": "lint-staged && npm run test:api && npm run test-docs",
      "pre-push": "npm test"
    }
  },
...
```

Reply:
Just adding it to `.github/workflows/taiko` is fine for now as these are functional tests. The husky pre-commit hook currently only runs unit tests.

Reply:
I tried several times, but it looks like I am not able... maybe I'll try again tomorrow.

Reply:
My impression is that CI/CD pipeline is broken for all PRs, even when I rolled back to `e12a2e0` which previously was ok...

![image](https://user-images.githubusercontent.com/53738017/94585782-c50d8880-0280-11eb-98bc-9f464def42d0.png)

I am not able to reproduce the same problem on my machine. So I don't know what to do about that,,,

Reply:
ok, I was able to reproduce it locally... opening an issue right away

Reply:
A fix was pushed for this, can you try rebasing and adding an entry to `.github/workflows/taiko`?

Reply:
I rebased to clean up the history which was getting somewhat contorted.

Then I applied to `docs-tests` similar changes to those made by BugDiver in `functional-tests`.

Finally, I am trying to add the new job to the github workflow 🤞

Reply:
All green 🎉